test/mobile/test_bytecode.py

```diff
@@ -15,33 +25,231 @@
+                model_path = pytorch_test_dri / "cpp" / "jit" / model_name
+                output_model_path_backport = Path(tmpdirname, "script_module_backport.ptl")
 
-        module_input = 1
-        jit_module_v4_result = jit_module_v4(module_input)
-        mobile_module_v4_result = mobile_module_v4(module_input)
+                current_to_version = current_from_version - 1
+                backport_success = _backport_for_mobile(model_path, output_model_path_backport, current_to_version)
+                assert(backport_success)
+
+                expect_bytecode_pkl = SCRIPT_MODULE_BYTECODE_PKL[current_to_version]["bytecode_pkl"]
+
+                buf = io.StringIO()
+                torch.utils.show_pickle.main(
+                    ["", tmpdirname + "/" + output_model_path_backport.name + "@*/bytecode.pkl"],
+                    output_stream=buf)
+                output = buf.getvalue()
+
+                acutal_result_clean = "".join(output.split())
+                expect_result_clean = "".join(expect_bytecode_pkl.split())
+                isMatch = fnmatch.fnmatch(acutal_result_clean, expect_result_clean)
+                assert(isMatch)
+
+                current_from_version -= 1
+        shutil.rmtree(tmpdirname)
+
+    def test_all_backport_functions(self):
+        # Backport from the latest bytecode version to the minimum support version
+        # Load, run the backport model, and check version
+        class TestModule(torch.nn.Module):
+            def __init__(self, v):
+                super().__init__()
+                self.x = v
+
+            def forward(self, y: int):
+                increment = torch.ones([2, 4], dtype=torch.float64)
+                return self.x + y + increment
+
+
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            output_model_path = Path(tmpdirname, "script_module.ptl")
+            script_module = torch.jit.script(TestModule(1))
+            optimized_scripted_module = optimize_for_mobile(script_module)
+            exported_optimized_scripted_module = optimized_scripted_module._save_for_lite_interpreter(str(output_model_path))
+
+            current_from_version = _get_model_bytecode_version(output_model_path)
+            current_to_version = current_from_version - 1
+            output_model_path_backport = Path(tmpdirname, "script_module_backport.ptl")
+            while current_to_version >= MINIMUM_TO_VERSION:
+                # Backport model to v4 to buffer
+                backport_success = _backport_for_mobile(output_model_path, output_model_path_backport, current_to_version)
+                assert(backport_success)
 
-        expected_result = 3 * torch.ones([2, 4], dtype=torch.float64)
+                backport_version = _get_model_bytecode_version(output_model_path_backport)
+                assert(backport_version == current_to_version)
+
+                buf = io.StringIO()
+                torch.utils.show_pickle.main(
```

Comment:
In the previous PR, is the get bytecode function exported to python too? show_pickle returns a stream. Just curious if get_bytecode would return a data structure (tuple) that can be used to query different tables and values.

Reply:
`get_bytecode_values()` is just a function at this moment. Yeah it will be helpful to expose it in python and cpp. I think this task can be merged with the task simple model cpp reader.